app/grandchallenge/algorithms/forms.py

```diff
@@ -103,13 +103,16 @@ def __init__(self, *args, algorithm, user, **kwargs):
             self.fields["algorithm_image"].initial = active_image
 
         for inp in self._algorithm.inputs.all():
-            self.fields[inp.slug] = InterfaceFormField(
-                instance=inp,
-                initial=inp.default_value,
-                user=self._user,
-                required=(inp.kind != InterfaceKindChoices.BOOL),
-                help_text=clean(inp.description) if inp.description else "",
-            ).field
+            if inp is not None:
```

Comment:
Just for my understanding - when would `None` be returned here?

Reply:
Sorry, this is a copy paste error!